src/importexport/midi/internal/midiconfiguration.cpp

```diff
@@ -44,6 +45,7 @@ void MidiConfiguration::init()
 
     settings()->setDefaultValue(EXPAND_REPEATS_KEY, Val(true));
     settings()->setDefaultValue(EXPORTRPNS_KEY, Val(true));
+    settings()->setDefaultValue(SPACELYRICS_KEY, Val(true));
```

Comment:
As this spacing didn't happen before, `false` would keep the old behavior

Reply:
As the Recommended Practice for SMF asks for the space, I'd argue in favor of making the default true under the "fixing a bug"-umbrella

Reply:
OK, fair enough. But then the question is why making it an option?

Reply:
I'd be personally fine with not having the option and just fixing the space behavior.
I'm unsure about how many other programs might rely on the "incorrect" (technically allowed, but not recommended by the standard) behavior we've had so far? In which case the option offers backwards compatibility..

Reply:
> OK, fair enough. But then the question is why making it an option?

Not all languages use whitespaces.